marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -245,9 +245,85 @@ void testRowExport() throws Exception {
             });
     }
 
-    private void assertRowsAndColumns(int limit, int totalColumns, String response) {
+    @Test
+    void testRowExportWithPathRangeQuery() throws Exception {
+        runAsDataHubDeveloper();
+        ReferenceModelProject project = installOnlyReferenceModelEntities(true);
+        deployEntityIndexes(getHubConfig());
```

Comment:
I believe dataHub.updateIndexes(); will work here too - can you try that? I recently used it in DeployProtectedPaths..Test and it worked well.

Reply:
dataHub isn't available to the hub central tests, unless I'm missing a function someplace. I believe the DataHub class is still depending on the auto-wiring of the HubConfig.